src/renderer/components/dock/logs/get-log-tab-data.injectable.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import logTabStoreInjectable from "./tab-store.injectable";
+
+const getLogTabDataInjectable = getInjectable({
+  instantiate: (di) => di.inject(logTabStoreInjectable).getData,
```

Comment:
So the logTabStore is not directly a dependency but its methods are? (just seeing if I understand)

Reply:
Yes that is correct. This is one of the benefits of DI since it makes testing much easier as each of these dependencies can not be overridden individually.

You can observe this in the modified tests for `log-resource-selector`